### ISSUE TITLE
TypeError: web3.toBigNumber is not a function

### ISSUE THREAD

**Issue by akvadrako** [ConsenSys-archive/ganache-cli-archive#476]
* I asked about this on [gitter](https://gitter.im/ConsenSys/truffle): [ ]

## Expected Behavior

Not throw an error

## Current Behavior

```
Forked Chain
==================
Location:    http://localhost:8545
/src/build/cli.node.js:99398
var Module;if(!Module)Module=(typeof Module!=="undefined"?Module:null)||{};var moduleOverrides={};for(var key in Module){if(Module.hasOwnProperty(key)){moduleOverrides[key]=Module[key]}}var ENVIRONMENT_IS_WEB=false;var ENVIRONMENT_IS_WORKER=false;var ENVIRONMENT_IS_NODE=false;var ENVIRONMENT_IS_SHELL=false;if(Module["ENVIRONMENT"]){if(Module["ENVIRONMENT"]==="WEB"){ENVIRONMENT_IS_WEB=true}else if(Module["ENVIRONMENT"]==="WORKER"){ENVIRONMENT_IS_WORKER=true}else if(Module["ENVIRONMENT"]==="NODE"){ENVIRONMENT_IS_NODE=true}else if(Module["ENVIRONMENT"]==="SHELL"){ENVIRONMENT_IS_SHELL=true}else{throw new Error("The provided Module['ENVIRONMENT'] value is not valid. It must be one of: WEB|WORKER|NODE|SHELL.")}}else{ENVIRONMENT_IS_WEB=typeof window==="object";ENVIRONMENT_IS_WORKER=typeof importScripts==="function";ENVIRONMENT_IS_NODE=typeof process==="object"&&"function"==="function"&&!ENVIRONMENT_IS_WEB&&!ENVIRONMENT_IS_WORKER;ENVIRONMENT_IS_SHELL=!ENVIRONMENT_IS_WEB&&!ENVIRONMENT_IS

TypeError: web3.toBigNumber is not a function
    at /src/build/webpack:/cli.js:232:1
    at Immediate._onImmediate (/src/build/webpack:/node_modules/ganache-core/lib/subproviders/geth_api_double.js:35:8)
    at runCallback (timers.js:789:20)
    at tryOnImmediate (timers.js:751:5)
    at processImmediate [as _immediateCallback] (timers.js:722:5)
```

## Possible Solution

?

## Steps to Reproduce (for bugs)

```
docker run trufflesuite/ganache-cli:latest --net=host --fork http://localhost:8545
```

## Context

Docker image doesn't work with --fork option.

## Your Environment
* Version used: latest
* Version of Truffle/Remix/Other tools used:
* NodeJS Version: [ ] 6.x, [ ] 7.x (unsupported), [ ] 8.x, [ ] 9.x
* Operating System and version (include distro if Linux):
* Link to your project or repro gist:
* Commit hash to use with above link for reproduction:
* I intend to submit a pull request to fix this issue: [ ]

**Comment by liangzan** [comment]
I encountered the same error. The error is due to web3. `toBigNumber` is a function in web3 v0.x versions. It was replaced by `toBN` in v1.x. It means it assumes it is calling the v0.x version of web3 when the v1.x version is in the scope. To resolve it, make sure the correct web3 is in scope.

**Comment by benjamincburns** [comment]
Thanks for reporting this, @akvadrako. This is fixed as of `ganache-cli@6.1.0`

**Comment by cliffhall** [comment]
> I encountered the same error. The error is due to web3. `toBigNumber` is a function in web3 v0.x versions. It was replaced by `toBN` in v1.x. It means it assumes it is calling the v0.x version of web3 when the v1.x version is in the scope. To resolve it, make sure the correct web3 is in scope.

That's actually [`web3.utils.toBN`](https://web3js.readthedocs.io/en/v1.2.0/web3-utils.html?highlight=toBN#tobn).
